src/components/controls/panelToggle.tsx

```diff
@@ -0,0 +1,31 @@
+// A slider toggle to adjust the state of a panel via dispatch.
+
+import React from "react";
+import { useAppDispatch } from "../../hooks";
+
+import Toggle from "./toggle";
+import { togglePanelDisplay } from "../../actions/panelDisplay";
+
+type Props = {
+  panel: string
+  on: boolean
+}
+
+const PanelToggle = ({ panel, on }: Props) => {
```

Comment:
The positioning of the toggle is currently so that the bottom of the toggle is in-line with the text baseline (perhaps by design, perhaps not). I think it's aesthetically nicer to align the (vertical) centers, similar to how other toggles are positioned in the sidebar. 
![image](https://github.com/nextstrain/auspice/assets/8350992/41943e94-dc63-4867-80a4-17d0c5f50746)

Reply:
This is something I noticed but after spending a long time inspecting the elements, I couldn't figure out how to fix it.

It's not aligned to the bottom, though it may look like that coincidentally with the font size of "Tree". It, specifically `ToggleBackground`, is aligned to the top of `HeaderContainer`.

![image](https://github.com/nextstrain/auspice/assets/13424970/6e00bf9e-9791-4050-8126-b5cfa6d5e33b)

I would like to get this fixed before merging. Let me know if you have any ideas.

Reply:
The following works, but you'll have to play with exactly how many pixels to use!

```diff
diff --git a/src/components/controls/toggle.js b/src/components/controls/toggle.js
index e0ba253a..c52fb2c6 100644
--- a/src/components/controls/toggle.js
+++ b/src/components/controls/toggle.js
@@ -7,6 +7,7 @@ const ToggleBackground = styled.label`
   display: inline-block;
   width: 35px;
   height: 21px;
+  margin-top: 3px;
   input {
     display: none;
   }
```

Reply:
Adding `margin-top` is more of a patch that doesn't address the alignment issue. I'll use it if we can't figure out how to align properly. I think I might have figured it out though: dc76d27311a6868cba97a83db93203c32c81765a...2198a244a41ba398ef7b2de05981253c168d39cb

Reply:
👍